### PR TITLE
Use arguments file to avoid long classpath. #402

### LINKED ISSUE #402: Powershell launcher errors on big classpath
### Expected behavior
Gauge java project should run successfully

### Actual behavior
Gauge-java's `launcher.ps1` tries to set the CLASSPATH as an environment variable, and for some projects this can exceed the limit set by the OS/shell and result in error such as reported here: https://spectrum.chat/gauge/general/could-not-find-or-load-main-class-com-thoughtworks-gauge-gaugeruntime~df3528dc-82a6-4418-b991-15271177fb32?m=MTU4NDk1Njc3NDIxNQ==

```
C:\Users\xxxxxxxxx\tools\gauge\gauge.exe run --machine-readable --hide-suggestion --simple-console --env abc C:/Users/xxxxxxxxx/IdeaProjects/automationframework\_new/specs/development/blahh/orderhandlingservice/BusinessWorkflow.spec:71
Der Umgebungsvariablenname oder -wert ist zu lang (Environment variable name or value too long)
In C:\Users\xxxxxxxxx\tools\gauge\plugins\java\0.7.4\launcher.ps1:91 Zeichen:5
- $env:CLASSPATH = $global:classpath
- ~~~~~~~~~~~~~~
- CategoryInfo : InvalidOperation: (:) [], ArgumentException
- FullyQualifiedErrorId : ProviderContentWriteError
Fehler: Hauptklasse com.thoughtworks.gauge.GaugeRuntime konnte nicht gefunden oder geladen werden
Ursache: java.lang.ClassNotFoundException: com.thoughtworks.gauge.GaugeRuntime
```

### Steps to reproduce
Reported by user : https://spectrum.chat/gauge/general/could-not-find-or-load-main-class-com-thoughtworks-gauge-gaugeruntime~df3528dc-82a6-4418-b991-15271177fb32?m=MTU4NDk1Njc3NDIxNQ==

### Dev notes

Instead of setting CLASSPATH environment variable, launch java using `-cp` flag.

### PR DESCRIPTION
**Pull request by BugDiver**
Fixes #402

### CODE REVIEW

**Comment by BugDiver** [comment]
> @BugDiver Thank you for contributing to gauge-java. Your pull request has been labeled as a release candidate 🎉🎉.
> 
> Merging this PR will trigger a release.
> 
> ### Please bump up the version as part of this PR.
> Instructions to bump the version can found at [CONTRIBUTING.md](https://github.com/getgauge/gauge-java/blob/master/CONTRIBUTING.md)
> 
> If the CONTRIBUTING.md file does not exist or does not include instructions about bumping up the version, please looks previous commits in git history to see what changes need to be done.

This won't be required as the version is already bumped up.

File: bin/launcher.ps1 [hunk 1]
```diff
@@ -100,12 +100,22 @@ $tasks.Add('start', {
...
-    $env:CLASSPATH = $global:classpath
     if ("$env:GAUGE_DEBUG_OPTS" -ne "" ) {
       $debugArgs = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=$env:GAUGE_DEBUG_OPTS,timeout=25000"
       Write-Output "`nRunner Ready for Debugging"
     }
-    & $javaCommand `"-Dfile.encoding=UTF-8`" $debugArgs $env:gauge_jvm_args com.thoughtworks.gauge.GaugeRuntime --start
+
+    $verison = (Get-Command java | Select-Object -ExpandProperty Version).toString() | Out-String
+
+    if ( "$verison" -lt "9" ) {
```
Comment (zabil) [inline bin/launcher.ps1]: This check should not be used to display that version 8 is not supported anymore.
Raise it all the way up to Gauge to inform the user to Upgrade Java or stick to older versions of the plugin

Comment (BugDiver) [reply bin/launcher.ps1:110]: @zabil 

```
➜  java-foo javac -version
javac 1.8.0_202
➜  java-foo gauge run specs
This version of the plugin does not support java version < 1.9
Please upgrade your java version or stick to an older version of gauge-java.
Error ----------------------------------

[Gauge]
Failed to start gauge API: Error occurred while waiting for runner process to finish.
Error : exit status 1

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
	darwin, 1.0.8, 3abcc41d
```

Comment (zabil) [reply bin/launcher.ps1]: Can you modify this to 

```
➜  java-foo gauge run specs

This version of gauge-java plugin does not support Java versions < 1.9
Please upgrade your Java version or use a version of gauge-java <= v0.7.4

Error ----------------------------------

[Gauge]
Failed to start gauge API: Error occurred while waiting for runner process to finish.
Error : exit status 1

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
	darwin, 1.0.8, 3abcc41d
```

**Pull request closed without merging**